scripts/datasets/extract_monocular_cues.py

```diff
@@ -152,23 +157,26 @@ def save_outputs(img_path, output_file_name):
         output = model(img_tensor).clamp(min=0, max=1)
 
         if args.task == "depth":
-            # output = F.interpolate(output.unsqueeze(0), (512, 512), mode='bicubic').squeeze(0)
+            if scale_factor > 1:
+                output = F.interpolate(output.unsqueeze(0), scale_factor=scale_factor, mode="bicubic").squeeze(0)
             output = output.clamp(0, 1)
 
             np.save(save_path.replace(".png", ".npy"), output.detach().cpu().numpy()[0])
             plt.imsave(save_path, output.detach().cpu().squeeze(), cmap="viridis")
         else:
+            if scale_factor > 1:
+                output = torch.nn.functional.interpolate(output, scale_factor=scale_factor, mode="nearest")
```

Comment:
Why use "nearest" for normal and "bicubic" for depth

Reply:
the depth value used bicubic from the original uncommented code, I tried using bicubic for normal map but it generated some artifacts around boundaries. I could also switch depth to nearest for consistency